src/python/pants/backend/python/lint/python_format_target.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from dataclasses import dataclass
+
+from pants.engine.legacy.structs import (
+  PantsPluginAdaptor,
+  PythonAppAdaptor,
+  PythonBinaryAdaptor,
+  PythonTargetAdaptor,
+  PythonTestsAdaptor,
+  TargetAdaptor,
+)
+from pants.engine.rules import UnionMembership, UnionRule, rule, union
+from pants.engine.selectors import Get, MultiGet
+from pants.rules.core.fmt import FmtResult, FmtResults, FormatTarget
+
+
+@union
+class PythonFormatTarget:
+  pass
+
+
+@dataclass(frozen=True)
+class ConcretePythonFormatTarget:
```

Comment:
I think that because this is private, you could maybe send a signal by prefixing it `_`: ie, `_ConcretePythonFormatTarget`.